zowe_conformance/terms_and_conditions.md

```diff
@@ -57,7 +75,7 @@ The Test Evaluation Guide contains details about the self-testing process and th
 
 ### 4. Submit Test Results
 
-After passing the designated tests and satisfying the applicable criteria, a Participant must submit to The Linux Foundation a copy of the testing results generated by the testing evaluation program. If the Zowe project has not made a testing evaluation program available, then for each potential Qualifying Offering, the Participant must affirm its conformance as set forth on the Participation Form or other system of record made available by The Linux Foundation. These results may be made available to the public on the Zowe project website or other public repository if Participant makes public use of the Conformance Marks. However, if the Participant decides not to proceed with any such public use following submission of testing results to The Linux Foundation, then upon Participant's written request to The Linux Foundation the test results will not be publicly disclosed, subject to sharing of test results with members of the Zowe project community as described in Section 5 below.
```

Comment:
@Joe-Winchester - is there a particular reason why this text was removed, or was it just an oversight...

`However, if the Participant decides not to proceed with any such public use following submission of testing results to The Linux Foundation, then upon Participant's written request to The Linux Foundation the test results will not be publicly disclosed, subject to sharing of test results with members of the Zowe project community as described in Section 5 below.`

Reply:
It looks like an oversight.  Thanks for picking up on this - I've added the sentences back into `4. Submit Test Results`